### ISSUE TITLE
azurerm_local_network_gateway doesn't send address_space

### ISSUE THREAD

**Issue by pergeri** [hashicorp/terraform-provider-azurerm#15109]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version
Terraform v1.1.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v2.93.1

### Affected Resource(s)

* `azurerm_local_network_gateway`

### Terraform Configuration Files

**Imported config:**
```hcl
resource "azurerm_local_network_gateway" "example" {
  name                = "example"
  resource_group_name = "rg-example"
  location            = "North Europe"
  gateway_address     = "1.2.3.4"
  address_space = [
    "10.205.2.9/32",
    "10.205.2.201/32",
    "10.205.2.9/32",
    "10.205.2.201/32"
  ]
}
```
**Planned config:**
```hcl
resource "azurerm_local_network_gateway" "example" {
  name                = "example"
  resource_group_name = "rg-example"
  location            = "North Europe"
  gateway_address     = "1.2.3.4"
  address_space = [
    "10.205.2.9/32",
    "10.205.2.201/32"
  ]
}
```

### Debug Output

[GIST](https://gist.github.com/pergeri/d65a239268fae9c56923fd051c3d2e66)
(some data was redacted since it was too much data for gist)
### Expected Behaviour
The Address prefixes in address_space should be updated. In this case two duplicated addresses should've been de-duplicated. 

### Actual Behaviour
Terraform tries to remove all address prefixes which azure does not allow since we do not set bgp settings. (line 1744 in gist)

**Error returned from Azure:**
```json
{
  "error": {
    "code": "LocalNetworkGatewayCannotHaveEmptyAddressPrefixAndBgpSettings",
    "message": "The local network gateway /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg-example/providers/Microsoft.Network/localNetworkGateways/example cannot have both empty address prefix and empty BGP Settings.",
    "details": []
  }
}
```

### Steps to Reproduce

1. Create a local network gateway in Azure with some address prefixes duplicated.
2. Create a terraform workspace and import as a `azurerm_local_network_gateway`.
3. Remove duplicated addresses from terraform config.
4. `terraform plan`
5. `terraform apply`, this triggers the error.

**Comment by sinbai** [comment]
@pergeri thanks for reporting this issue. PR has been submitted to fix it.

**Comment by stefan-rapp** [comment]
Any updates?

**Comment by Mr-Silv** [comment]
Any updates on this?

**Comment by WagnerJanissetti** [comment]
Any update on this? PR is still opened waiting for review

**Comment by katbyte** [comment]
Closing as it appears this has been fixed by #15159
